Mathlib/FieldTheory/RatFunc.lean

```diff
@@ -814,6 +814,8 @@ instance (R : Type*) [CommSemiring R] [Algebra R K[X]] : Algebra R (RatFunc K) w
         ofFractionRing_mul, IsLocalization.mul_mk'_eq_mk'_of_mul, Algebra.smul_def]
   commutes' c x := mul_comm _ _
 
+instance : Coe (Polynomial K) (RatFunc K) := ⟨fun P ↦ algebraMap _ _ P⟩
```

Comment:
Could you define a separate function? That's required for the `@[coe]` annotation which we need for certain tactics. Something like (untested):
```suggestion
@[coe]
def coePolynomial (P : Polynomial K) : RatFunc K := algebraMap _ _ P

instance : Coe (Polynomial K) (RatFunc K) := ⟨coePolynomial⟩
```

Reply:
Done (and it worked)